utils/tfw_logger.cc

```diff
@@ -0,0 +1,463 @@
+/**
+ *		Tempesta FW
+ *
+ * Copyright (C) 2024 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+#include <fcntl.h>
+#include <signal.h>
+#include <spdlog/sinks/basic_file_sink.h>
+#include <spdlog/spdlog.h>
+#include <stdio.h>
+#include <unistd.h>
+
+#include <fstream>
+#include <future>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+
+#include <boost/program_options.hpp>
+#include <clickhouse/base/socket.h>
+#include <clickhouse/client.h>
+
+#include "../fw/access_log.h"
+#include "clickhouse.hh"
+#include "mmap_buffer.hh"
+#include "error.hh"
+
+namespace po = boost::program_options;
+
+#define FILE_PATH	"/dev/tempesta_mmap_log"
+#define TABLE_NAME	"access_log"
+
+constexpr char pid_file_path[] = "/var/run/tfw_logger.pid";
+constexpr size_t WAIT_FOR_FILE = 1;  /* s */
```

Comment:
```suggestion
constexpr std::chrono::seconds WAIT_FOR_FILE = 1; 
```

Reply:
Thus you avoid memorizing the units of measurements and need in commenting it

Reply:
Fixed.